### ISSUE TITLE
Feature Idea: Filter & Tags

### ISSUE THREAD

**Issue by syedsimanta03** [cassidoo/next-netlify-blog-starter#7]
Hi there, Blog posts should have search bar/option, user should be able to filter post by clicking on tags like *react* tag shows react posts

**Comment by TechStacker** [comment]
Yeah, I've been looking for months for a way to sort/filter markdown posts in Next.js, based on tags or categories (defined in the .md file). No luck!
